pkg/deploy/generator/scripts/gatewayVMSS.sh

```diff
@@ -67,6 +72,18 @@ include /etc/logrotate.d
     create 0600 root utmp
     rotate 1
 }
+
+# Maximum log directory size is 100G with this configuration
+# Setting limit to 100G to allow space for other logging services
```

Comment:
please update this comment to match the size on line 80

Reply:
The comment is correct, it's a multiple of `size` and `rotate` options. 5 logs of size 20G will be kept.